src/view/plandetail/PlaceInfoTab.tsx

```diff
@@ -125,14 +127,22 @@ export const TabPanel = ({
 const TabPanelInformation = ({
     categories,
     priceRange,
+    googlePlaceReviews,
 }: {
     categories: PlaceCategory[];
     priceRange: PriceRange | null;
+    googlePlaceReviews: GooglePlaceReview[] | null;
 }) => {
     const isCategoryEmpty = categories.length === 0;
     const isPriceRangeEmpty = !priceRange || priceRange.max === 0;
+    const reviews = googlePlaceReviews;
+    const isGooglePlaceReviews = !reviews || reviews.length === 0;
+    const totalStars = googlePlaceReviews.reduce((acc, review) => {
+        return acc + review.rating;
+    }, 0);
+    const averageStars = totalStars / googlePlaceReviews.length;
```

Comment:
スマートに`reduce`つかえて良さげです！
ここの処理がstarの平均を求めるためのひとかたまりの処理だよ、とわかるように`average`関数とかつくってあげると良いかも。

```suggestion
    const averageStars = average(googlePlaceReviews.map((review) => review.rating));
    
    const average = (values: number[]) => {
       const sum =  reduce((sum, review) => acc + review.rating;, 0);
       return sum / values.length;
    }
```

Reply:
[imo] 
これまったく強い意志は無いんですけど、`averageRating`のほうがデータに従っているのかな？と思いました。
startだと、UIに従っている感がしてます。
コメント程度に思ってください。